### DISCUSSION TITLE
Interested in GSOC2024 Project: Synthetic Anomaly Generation

### DISCUSSION THREAD

**Discussion by JoannaCCJH** [open-edge-platform/anomalib#1764] [General]
Hi! My name is Joanna, and I am currently a sophomore student majoring in Computer Science. I want to participate in GSOC, and I am interested in the Synthetic Anomaly Generation for Threshold Selection and Model Evaluation project.  I would appreciate some guidance on where to start with this project. Also, could someone give me some suggestions on which feature on task under "Issues" to start with, please?

**Comment by blaz-r** [comment]
Hi, it's great to see interest in GSoC at Anomalib ☺️. The mentors will be able to tell you more about project specifics. Meanwhile, many tasks can be tackled. Check [good first issues](https://github.com/openvinotoolkit/anomalib/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Issue%22) and possibly other issues/feature requests.

@adrianboguszewski could you maybe just clarify if the issue prerequisite can be on any repository inside Openvinotoolkit, or only the selected from [Good first issues](https://github.com/orgs/openvinotoolkit/projects/3/views/1). In any case, it's a good idea to check out Anomalib issues to familiarize yourself with the codebase 😄.

**Reply by JoannaCCJH** [reply]
Thanks for replying! I will check out these good first issues!

**Reply by adrianboguszewski** [reply]
I talked to Samet and we're ok to use good first issues from Anomalib as a prerequisite task :)

**Reply by danylo-boiko** [reply]
@adrianboguszewski, cool, thanks a lot!

**Reply by JoannaCCJH** [reply]
Thanks for replying!

**Reply by samet-akcay** [reply]
I'll label more issues as `Good First Issue`

**Reply by RitikaxShakya** [reply]
@samet-akcay @djdameln @ashwinvaidya17 I've sent my draft proposal to your email, hopefully it doesnt go to spam folder. I would appreciate it if you could give me some suggestions though I've submitted it to the GSoC portal, but it can be changed if you have some suggestion in changing something :) Thank you very much!

**Comment by adrianboguszewski** [comment]
Hey @JoannaCCJH, please move it to the Google Summer of Code category as I follow only that one. 

The mentors for this project are @samet-akcay @djdameln @ashwinvaidya17

**Reply by JoannaCCJH** [reply]
ok!
